101-aks-azml-targetcompute/azuredeploy.json

```diff
@@ -0,0 +1,171 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+         "clusterName": {
+            "type": "string",
+            "defaultValue":"aksazmlcluster",
+            "metadata": {
+                "description": "The name of the Managed Cluster resource."
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "The location of the Managed Cluster resource."
+            }
+        },
+        "osDiskSizeGB": {
+            "type": "int",
+            "defaultValue": 100,
+            "metadata": {
+                "description": "Disk size (in GB) to provision for each of the agent pool nodes. This value minimun 30 to 1023. Specifying 100 is the default value required to be attached to Azure ML."
+            },
+            "minValue": 30,
+            "maxValue": 1023
+        },
+        "agentCount": {
+            "type": "int",
+            "defaultValue": 3,
+            "metadata": {
+                "description": "The number of nodes for the cluster."
+            },
+            "minValue": 1,
+            "maxValue": 50
+        },
+        "podCidr": {
+            "type": "string",
+            "defaultValue": "10.244.0.0/16",
+            "metadata": {
+                "description": "A CIDR notation IP range from which to assign pod IPs when kubenet is used"
+            }
+        },
+        "serviceCidr": {
+            "type": "string",
+            "defaultValue": "10.0.0.0/16",
+            "metadata": {
+                "description": "A CIDR notation IP range from which to assign service cluster IPs."
+            }
+        },
+        "dnsServiceIP": {
+            "type": "string",
+            "defaultValue": "10.0.0.10",
+            "metadata": {
+                "description": "An IP address assigned to the Kubernetes DNS service"
+            }
+        },
+        "dockerBridgeCidr": {
+            "type": "string",
+            "defaultValue": "172.17.0.1/16",
+            "metadata": {
+                "description": "A specific IP address and netmask for the Docker bridge, using standard CIDR notation."
+            }
+        },
+        "agentVMSize": {
+            "type": "string",
+            "defaultValue": "Standard_D3_v2",
+            "metadata": {
+                "description": "The size of the VM instances."
+            }
+        },
+        "kuberneteVersion": {
+            "type": "string",
+            "defaultValue": "1.16.9",
+            "metadata": {
+                "description": "Supported Azure Kubernetes version"
+            }
+        },
+        "linuxAdminUsername": {
+            "type": "string",
+            "defaultValue": "azureuser",
```

Comment:
remove defaultValue

Reply:
default values are the conditions that will help users build the target compute compliant with the requirements from Azure ML.

Reply:
If ML requires a specific user name, use a variable.  If it doesn't remove the defaultValue.